### DISCUSSION TITLE
Mutual Information?

### DISCUSSION THREAD

**Discussion by benjamin-cash** [dtcenter/METplus#1917] [Statistical Computation] [answered]
Do the METplus diagnostics include anything like Mutual Information, Transfer Entropy, or other similar quantities that come out of information theory?

**Comment by JohnHalleyGotway** [comment]
Hi @benjamin-cash, I did some google about mutual information and transfer entropy and believe that the answer is no. I'm not aware of any output from MET or METplus that would fall into these categories.

And it not clear to me what those would be. Would these be new statistics computed in the comparison of forecast and observation data in the MET tools? Or would this be something addressed via a new METplus use case or perhaps a python derivation or plotting functionality in METcalcpy or METplotpy?

If you have more followup questions, we could pull our resident statistician @ericgilleland into the conversation to see if he has any perspectives to share.

**Comment by ericgilleland** [comment, marked as answer]
We don't specifically have anything on mutual information in METplus.  It's
a different kind of philosophy than the type of verification we have in
there.  It is interesting stuff, though.  We might have something like
Shannon entropy in there, but I don't think we have it specifically.

**Comment by jprestop** [comment]
Hi @benjamin-cash.  I just wanted to follow up and see if you had any further questions.  If not, I’d like to mark this discussion as being answered and “lock it” to prevent future posts. That’s how we encourage users to create new discussions for new questions. But I wanted to give you an opportunity to comment on it before doing so. Please feel free to select one of the responses as being the best answer to your original question. That’ll help future users with similar questions find answers.

**Comment by benjamin-cash** [comment]
Hi @jprestop - No further questions, thanks for checking. :)
